app/controllers/sessions_controller.rb

```diff
@@ -1,12 +1,17 @@
 class SessionsController < ApplicationController
-  skip_before_action :authenticate_agent!, only: [:create]
+  skip_before_action :authenticate_agent!, only: [:create, :new]
   wrap_parameters false
   respond_to :json, only: :create
+  layout "website"
 
   before_action :retrieve_agent!, :mark_agent_as_logged_in!,
                 :set_agent_return_to_url,
                 only: [:create]
 
+  def new
+    redirect_to root_path if logged_in?
```

Comment:
Je me suis trompé sur ma dernière suggestion il vaut mieux
```suggestion
    redirect_to authenticated_root_path if current_agent
```
sinon ça va rediriger vers le formulaire de login sans avoir clear la session auparavant (ce qui n'est pas très grave à priori mais moins clean)